src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateFactory.java

```diff
@@ -9,8 +9,15 @@ private EmailTemplateFactory() {
     }
 
     public static AbstractEmailTemplate get(String bundle, String application) {
-        if (bundle.toLowerCase().equals("rhel") && application.toLowerCase().equals("policies")) {
-            return new Policies();
+        if (bundle.toLowerCase().equals("rhel")) {
+            switch (application.toLowerCase()) {
+                case "policies":
+                    return new Policies();
+                case "advisor":
+                    return new Advisor();
+                default:
```

Comment:
Sonar won't like the absence of `break;`.

Reply:
Added the break;